image-loader/app/model/UploadStatus.scala

```diff
@@ -0,0 +1,20 @@
+package model
+
+import play.api.libs.json._
+
+case class UploadStatus(
+                         id: String,
+                         uploadedBy: String,
+                         fileName: String,
```

Comment:
Both `uploadedBy` and `fileName` would be better modelled as `Option[String]`. There are also some missing fields here for `identifiers` and `uploadTime` - which again can both be `Option[String]`.